app/phpunit.xml.dist

```diff
@@ -8,7 +8,7 @@
     convertErrorsToExceptions   = "true"
     convertNoticesToExceptions  = "true"
     convertWarningsToExceptions = "true"
-    processIsolation            = "false"
+    processIsolation            = "true"
```

Comment:
@dennisameling this is amazing. Thank you for your hard work in areas where it's not visible for users but moves the whole project to another level!

We are currently working on speeding up the test suite. So far we were able to reduce the run time from ~20 minutes to ~8 minutes. Also using less memory. The community PR will come soon. However, this change is slowing the tests down by more than 1/3. Compare Travis for this PR:

https://travis-ci.org/github/mautic/mautic/builds/732687539

with another PR:

https://travis-ci.org/github/mautic/mautic/builds/731009039

I know you mean it as a temporary change but the plugin removal will be a BC break so it will have to wait for Mautic 4 and no one is working on the marketplace.

So I would suggest to run only the problematic tests in parallel. It can be done by adding this class annotation:
```php
/**
 * @runTestsInSeparateProcesses
 * @preserveGlobalState disabled
 */
```